### DISCUSSION TITLE
https://mnfienen.github.io/talulat/2023/07/13/hello_world.html

### DISCUSSION THREAD

**Discussion by giscus** [mnfienen/talulat#2] [Comments]
# https://mnfienen.github.io/talulat/2023/07/13/hello_world.html

We are going for a ride! miles today: 0 total miles: 0

https://mnfienen.github.io/talulat/2023/07/13/hello_world.html

**Comment by mnfienen** [comment]
are comments working?

**Reply by CMillerFienen** [reply]
This is a test . . . are you ready for CANADA????

**Reply by mnfienen** [reply]
LFG!!!!!!
